src/xdocs/checks/metrics/cyclomaticcomplexity.xml

```diff
@@ -22,7 +22,7 @@
           The complexity is equal to the number of decision points <code> + 1</code>.
           Decision points: <code>if</code>, <code>while</code>
           , <code>do</code>, <code>for</code>, <code>?:</code>, <code>catch</code>
-          , <code>switch</code>, <code>case</code>
+          , <code>switch</code>, <code>case</code>, <code>when</code>
           statements and operators <code>&amp;&amp;</code> and <code>||</code>
```

Comment:
> when statements

https://docs.oracle.com/javase/specs/jls/se22/html/jls-14.html#jls-Guard
> may have an optional when expression, known as a guard

This is not a statement.

Reply:
done, how do you feel about this being a list? I think this is more readable

Reply:
List would definitely be better here.

Reply:
ok done it is list now